SwiftMonkeyPaws/MonkeyPaws.swift

```diff
@@ -35,10 +31,60 @@ private let circleRadius: CGFloat = 7
     }
     ```
 */
+
 public class MonkeyPaws: NSObject, CALayerDelegate {
+
+    public struct Configuration {
```

Comment:
Would it make sense to move this `struct` to a separate file? If you want the usage to stay the same (`MonkeyPaws.Configuration`) it can be added as an extension:

```swift
extension MonkeyPaws {
    public struct Configuration {
        ...
    }
}
```

What do you think?